cluster_generator/utils.py

```diff
@@ -73,6 +258,101 @@ def generate_particle_radii(r, m, num_particles, r_max=None, prng=None):
     return radius, mtot
 
 
+def build_yt_dataset_fields(grid, models, domain_dimensions, centers, velocities):
```

Comment:
@Eliza-Diggins this should probably get its own file--we could probably put other ones in there down the line

Reply:
@jzuhone I've moved this to the new ``data_structures.py`` file.  Not sure if its a great name for the module though...

It also looks like it's currently unused; the intention here is to have a ``create_dataset`` method on both ``ClusterICs`` and ``ClusterModel``, but I think I've saved that for a future PR to avoid making any major changes to the core package in this PR (black and flake8 already did enough!)